modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable camelcase */
```

Comment:
Why is this in the deprecated folder?

Reply:
I decided to put it there because this [extension is deprecated](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_feature_metadata#%EF%B8%8F-ext_feature_metadata-has-been-replaced-by-ext_mesh_features-%EF%B8%8F) in docs and will be replaced with [EXT_mesh_features](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features) extension.